src/lager_app.erl

```diff
@@ -200,7 +221,10 @@ get_env_default({ok, Value}, _Default) ->
 
 start(_StartType, _StartArgs) ->
     {ok, Pid} = lager_sup:start_link(),
+    SavedHandlers = boot(),
+    {ok, Pid, SavedHandlers}.
 
+boot() ->
```

Comment:
Let's make `boot/1` that takes a Sink argument. `boot/0` should call `boot/1` with the argument `?DEFAULT_SINK`